delphi/translators/GrFN2WiringDiagram/autotranslated_WD_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
@pauldhein Could you please clear the outputs of the notebook and then re-add and commit it?